<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDialog</class>
 <widget class="QDialog" name="SettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>355</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Paramètres</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Ressources/icons/icons8-settings-96.png</normaloff>:/Ressources/icons/icons8-settings-96.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Observateur</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Latitude</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,1,6,1,6,1,1">
          <item>
           <widget class="QSpinBox" name="LatDegreeSpinBox">
            <property name="maximum">
             <number>90</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="LatMinuteSpinBox">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="LatSecondDoubleSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="maximum">
             <double>59.990000000000002</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="LatComboBox">
            <item>
             <property name="text">
              <string>N</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>S</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Longitude</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,1,6,1,6,1,1">
          <item>
           <widget class="QSpinBox" name="LonDegreeSpinBox">
            <property name="maximum">
             <number>180</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="LonMinuteSpinBox">
            <property name="maximum">
             <number>59</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>'</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="LonSecondDoubleSpinBox">
            <property name="locale">
             <locale language="English" country="UnitedStates"/>
            </property>
            <property name="maximum">
             <double>59.990000000000002</double>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>&quot;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="LonComboBox">
            <item>
             <property name="text">
              <string>E</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>O</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Altitude</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
          <item>
           <widget class="QSpinBox" name="AltitudeSpinBox">
            <property name="minimum">
             <number>-500</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>m</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Affichage</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Thème</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="LightRadioButton">
        <property name="text">
         <string>Light</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="DarkRadioButton">
        <property name="text">
         <string>Dark</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Base de données</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Chemin de la base de données</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLineEdit" name="DBPathLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="DBBrowsePushButton">
          <property name="text">
           <string>Explorer</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="SavePushButton">
       <property name="text">
        <string>Enregistrer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelPushButton">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
